vertical-pod-autoscaler/e2e/v1beta2/common.go

```diff
@@ -357,9 +357,27 @@ func getVpaClientSet(f *framework.Framework) vpa_clientset.Interface {
 
 // InstallVPA installs a VPA object in the test cluster.
 func InstallVPA(f *framework.Framework, vpa *vpa_types.VerticalPodAutoscaler) {
+	status := vpa.Status
 	vpaClientSet := getVpaClientSet(f)
-	_, err := vpaClientSet.AutoscalingV1beta2().VerticalPodAutoscalers(f.Namespace.Name).Create(context.TODO(), vpa, metav1.CreateOptions{})
+	vpa, err := vpaClientSet.AutoscalingV1beta2().VerticalPodAutoscalers(f.Namespace.Name).Create(context.TODO(), vpa, metav1.CreateOptions{})
 	gomega.Expect(err).NotTo(gomega.HaveOccurred(), "unexpected error creating VPA")
+	// apiserver ignore status in vpa create, so need to update status
+	if !isStatusEmpty(&status) {
+		vpa.Status = status
+		_, err := vpaClientSet.AutoscalingV1beta2().VerticalPodAutoscalers(f.Namespace.Name).UpdateStatus(context.TODO(), vpa, metav1.UpdateOptions{})
+		gomega.Expect(err).NotTo(gomega.HaveOccurred(), "unexpected error update VPA status")
+	}
```

Comment:
Please:

- Extract helper for patching status, make lines 399-406 a function
- Use the new helper to patch status `PatchVpaRecommendation` and `InstallVPA` (then we don't need to make a copy of created VPA)